diff --git a/src/Services/FIleService/FIleService.Presentation/Program.cs b/src/Services/FIleService/FIleService.Presentation/Program.cs
index f33f710..a8c9101 100644
--- a/src/Services/FIleService/FIleService.Presentation/Program.cs
+++ b/src/Services/FIleService/FIleService.Presentation/Program.cs
@@ -1,57 +1,14 @@
-using FIleService.Application.Hubs;
-using FIleService.Application.Interfaces;
-using FIleService.Application.Services;
-using Microsoft.OpenApi.Models;
+using FIleService.Presentation.Extensions;
 
 var builder = WebApplication.CreateBuilder(args);
-builder.Services.AddScoped<IFileGeneratorService, FileGeneratorService>();
-builder.Services.AddScoped<IFileMergeService, FileMergeService>();
-builder.Services.AddEndpointsApiExplorer();
-builder.Services.AddControllers();
-builder.Services.AddSwaggerGen(options =>
-{
-    options.AddSecurityDefinition(
-        "Bearer",
-        new OpenApiSecurityScheme
-        {
-            Description = @"Enter JWT Token please.",
-            Name = "Authorization",
-            In = ParameterLocation.Header,
-            Type = SecuritySchemeType.Http,
-            BearerFormat = "JWT",
-            Scheme = "Bearer"
-        }
-    );
-    options.AddSecurityRequirement(
-        new OpenApiSecurityRequirement
-        {
-            {
-                new OpenApiSecurityScheme
-                {
-                    Reference = new OpenApiReference
-                    {
-                        Type = ReferenceType.SecurityScheme,
-                        Id = "Bearer"
-                    },
-                },
-                new List<string>()
-            }
-        }
-    );
-});
 
-var app = builder.Build();
-app.MapHub<ImportProgressHub>("/importProgressHub");
-
-if (app.Environment.IsDevelopment())
-{
-    app.UseSwagger();
-    app.UseSwaggerUI();
-}
+builder.AddDatabase();
+builder.AddServices();
+builder.AddSwaggerDocumentation();
 
-app.UseHttpsRedirection(); 
-app.UseRouting(); 
+var app = builder.Build();
 
-app.MapControllers();
+app.AddSwagger();
+app.AddApplicationMiddleware();
 
 app.Run();
\ No newline at end of file
diff --git a/src/Services/FIleService/FIleService.Presentation/appsettings.Development.json b/src/Services/FIleService/FIleService.Presentation/appsettings.Development.json
index 0c208ae..d81c534 100644
--- a/src/Services/FIleService/FIleService.Presentation/appsettings.Development.json
+++ b/src/Services/FIleService/FIleService.Presentation/appsettings.Development.json
@@ -4,5 +4,8 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
+  },
+  "ConnectionStrings": {
+    "ConnectionString": "Host=localhost;Port=5432;Database=file_service;Username=postgres;Password=CHEATS145"
   }
 }
